documentation/prometheus-mixin/alerts.libsonnet

```diff
@@ -128,6 +128,12 @@
             alert: 'PrometheusNotIngestingSamples',
             expr: |||
               rate(prometheus_tsdb_head_samples_appended_total{%(prometheusSelector)s}[5m]) <= 0
+              and
+              (
+                sum without(scrape_job) (prometheus_target_metadata_cache_entries{%(prometheusSelector)s}) > 0
+                or
+                sum without(rule_group) (prometheus_rule_group_rules{%(prometheusSelector)s}) > 0
+              )
```

Comment:
I hope this will all solved by `promfmt` soon, but the mysterious indentation rules I use here desire the following formatting:
```PromQL
              (
                rate(prometheus_tsdb_head_samples_appended_total{%(prometheusSelector)s}[5m]) <= 0
              and
                (
                  sum without(scrape_job) (prometheus_target_metadata_cache_entries{%(prometheusSelector)s}) > 0
                or
                  sum without(rule_group) (prometheus_rule_group_rules{%(prometheusSelector)s}) > 0
                )
              )
```